students/ryba_paula/lesson_12_debugging/debugging_coin_toss.py

```diff
@@ -0,0 +1,22 @@
+import random
+import logging
+logging.basicConfig(level=logging.DEBUG, format=' %(asctime)s - %(levelname)s '
+                                                '- %(message)s')
+
+guess_array = ['heads', 'tails']
+guess = ''
+toss = random.choice(guess_array)
+logging.debug(toss)
+
+while guess not in guess_array:
+    guess = input('Guess the coin toss! Enter heads or tails: ')
+
```

Comment:
Można by jeszcze dodać logowanie danych wprowadzonych przez użytkownika. (to samo w linii 18)